lambdas/webhook.py

```diff
@@ -124,42 +113,62 @@ def valid_signature(event: dict) -> bool:
     return request_signature == expected_signature
 
 
-def handle_pod_export_webhook(message_body: dict) -> dict[str, object]:
+def handle_job_end_webhook(message_body: dict) -> dict:
+    job_name = message_body["job_instance"]["name"]
+    if job_name == os.environ["ALMA_POD_EXPORT_JOB_NAME"]:
+        logger.info("PPOD export job webhook received.")
+        job_type = "PPOD"
+    elif job_name.startswith(os.environ["ALMA_TIMDEX_EXPORT_JOB_NAME_PREFIX"]):
+        logger.info("TIMDEX export job webhook received.")
+        job_type = "TIMDEX"
+    else:
+        logger.info(
+            "POST request received and validated, no action triggered. Returning 200 "
```

Comment:
Is there any utility to being more descriptive in the `else` log message? Not being terribly familiar with the workflow, I'm not sure if more info would be useful in troubleshooting why the `job_name` isn't lining up with POD or TIMDEX